src/main/java/pl/betoncraft/betonquest/conversation/InventoryConvIO.java

```diff
@@ -142,6 +149,21 @@ public void display() {
 		npcMeta.setOwner(npcName);
 		npcMeta.setDisplayName(npcNameColor + npcName);
 		npcMeta.setLore(stringToLines(response, npcTextColor, null));
+		CitizensConversation citizensConv = (CitizensConversation) conv;
+		String texture = citizensConv.getNPC().data().get(NPC.PLAYER_SKIN_TEXTURE_PROPERTIES_METADATA);
+		if (texture != null) { // Can be null if not cached yet
+			GameProfile profile = new GameProfile(UUID.randomUUID(), null);
+			Property property = new Property("textures", texture, citizensConv.getNPC().data().get(NPC.PLAYER_SKIN_TEXTURE_PROPERTIES_SIGN_METADATA));
+			profile.getProperties().put("textures", property);
+
+			try {
+				Field field = npcMeta.getClass().getDeclaredField("profile");
+				field.setAccessible(true);
+				field.set(npcMeta, profile);
+			} catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException e) {
+				e.printStackTrace();
```

Comment:
Not everyone knows about that @Namnodorel. And since it's impossible to opt-out from Citizens skin textures we don't want to spam the console with errors. It will only make some people angry. The ones who have a problem will simply turn debugging on and see what's causing it or ask here or in the Spigot thread.